sljit_src/sljitNativePPC_common.c

```diff
@@ -652,7 +650,7 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 
 	compiler->error = SLJIT_ERR_COMPILED;
 	compiler->executable_offset = executable_offset;
-	compiler->executable_size = (sljit_uw)(code_ptr - code) * sizeof(sljit_ins);
+	compiler->executable_size = (sljit_uw)(code_ptr - code) * sizeof(sljit_ins) + trampoline_size;
```

Comment:
I would just move this after sljit_set_function_context

Reply:
but if you do, then have to track the 1 instruction padding independently of trampoline size